Project.toml

```diff
@@ -1,25 +1,29 @@
 name = "AbstractFFTs"
 uuid = "621f4979-c628-5d54-868e-fcf4e3e8185c"
-version = "1.5.0"
+version = "1.6.0"
 
 [deps]
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
```

Comment:
I think it would be better to not support this on Julia < 1.9 (possibly - separately - we could remove support for Julia < 1.10) than adding ForwardDiff as a hard dependency on Julia < 1.9.

Reply:
I personal think it's best to drop support for Julia <v1.10 but I guess that should be done in a separate PR.

Do you agree with the extension being here and not in ForwardDiff.jl? (I don't remember the reason I chose here.)

Reply:
One option is to add `ForwardDiff` only as a weak dep, and subsequently change the compat bound. To me having this extension in this package seems fine. Intuitively I tend to think that the package that defines the functions should be the one to own the extension, as the maintainers are probably the most familiar with the behavior of these functions.

Reply:
Yes, you can just remove this line and disable tests on Julia < 1.9:
```suggestion
```